packages/runtime/src/events/transport/IdentityDeletionProcessStatusChangedEvent.ts

```diff
@@ -0,0 +1,10 @@
+import { IdentityDeletionProcessDTO } from "../../types/transport/IdentityDeletionProcessDTO";
+import { DataEvent } from "../DataEvent";
+
+export class IdentityDeletionProcessStatusChangedEvent extends DataEvent<IdentityDeletionProcessDTO> {
+    public static readonly namespace = "tansport.identityDeletionProcessStatusChangedEvent";
+
+    public constructor(eventTargetAddress: string, data: IdentityDeletionProcessDTO) {
```

Comment:
Just a question for my own understanding: What's the point and purpose of `eventTargetAddress`?